sdk/appconfig/azappconfig/CHANGELOG.md

```diff
@@ -1,16 +1,10 @@
 # Release History
 
-## 0.3.0 (Unreleased)
-
-### Features Added
+## 0.3.0 (2022-05-10)
```

Comment:
I'm hoping to release this today once `azcore` release is done.

Reply:
We sure can! In general, Tuesday is the release date, but if releasing today is good for our customers, we should do it! I'll make a suggestion edit to change the date to 5/9, and I'll keep monitoring for the appearance of the Core release. Then, please approve this PR, and you can even submit the PR proposal with the date, and I'll trigger the release pipelines.